### DISCUSSION TITLE
Question: How to redirect UI event?

### DISCUSSION THREAD

**Discussion by HppZ** [microsoft/microsoft-ui-xaml#7450] [Q&A]
How to redirect event (such as ScrollEvent) to ListView even my pointer is not on ListView?

**Comment by StephenLPeters** [comment]
Can you describe your scenario? I'm not sure there is a way to get the actual event to be handled by an element out side of its routed event path, but I'm also not sure that would be neccisary.

**Comment by HppZ** [comment]
I want to scroll ListView when my mouse pointer is in green area.
restyle ListView is a solution but not the best one.

![image](https://user-images.githubusercontent.com/12285582/128586570-4ad218da-7e2d-4a6b-934e-5935c81163d4.png)

**Comment by yoshiask** [comment]
Is there any reason you can't stretch the ListView across the entire width, but keep the items in the center?

**Comment by StephenLPeters** [comment]
You could do what yoshiask suggests or include a second scrollviewer and bind the offset of the second to the first.

**Comment by StephenLPeters** [comment]
@RBrid are there are options for HppZ?

**Comment by RBrid** [comment]
I suppose you want to handle mouse wheel input over the green zone.  One option would be to handle the UIElement.PointerWheelChanged event for the green element and call ScrollViewer.ChangeView in response.

**Comment by HppZ** [comment]
> I suppose you want to handle mouse wheel input over the green zone. One option would be to handle the UIElement.PointerWheelChanged event for the green element and call ScrollViewer.ChangeView in response.

not smooth scrolling.

**Comment by HppZ** [comment]
> You could do what yoshiask suggests or include a second scrollviewer and bind the offset of the second to the first.

not smooth.
